test/unit/rules/lint-no-inline-styles-test.js

```diff
@@ -36,7 +36,20 @@ generateRuleTests({
     'style={{foo}}',
     'style="{{foo}} {{bar}}"',
     'style',
-    'style=""',
     'style="color:blue;margin-left:30px;"',
-  ].map(mapBadStyleExampleToTestRule),
+  ]
+    .map(mapBadStyleExampleToTestRule)
```

Comment:
You might want to see https://github.com/ember-template-lint/ember-template-lint/pull/873/commits/c4e29f6360d60e5f6fd9d63fcaad8d01dc1acb88 commit message